src/lib/addons/webhook.ts

```diff
@@ -29,9 +30,20 @@ export default class Webhook extends Addon {
             body = JSON.stringify(event);
         }
 
+        let headers;
+
+        if (authorization) {
+            headers = {
+                'Content-Type': contentType || 'application/json',
+                Authorization: authorization,
+            };
+        } else {
+            headers = { 'Content-Type': contentType || 'application/json' };
+        }
+
```

Comment:
```suggestion
        let headers = { 'Content-Type': contentType || 'application/json' }
        if (authorization) {
          headers = { Authorization: authorization, ...headers };
        }

```

Reply:
A matter of preference. Original is perfectly fine, or you can do something like this:
```ts
        const requestOpts = {
            method: 'POST',
            headers: {
                'Content-Type': contentType || 'application/json',
                Authorization: authorization || undefined,
            },
            body,
        };
```

Reply:
Ah, much better ways of doing this! 
Did I mess up in 
`{
                'Content-Type': contentType || 'application/json',
                Authorization: authorization,
            };`
by not putting Authorization key name in quotes?